jetson/teleop/src/__main__.py

```diff
@@ -210,6 +217,26 @@ def send_sa_kill(self):
         foot_msg.microscope_triad = 0
         lcm_.publish('/foot_openloop_cmd', foot_msg.encode())
 
+    def joint_b_calibration_callback(self, channel, msg):
+        if JointBCalibration.decode(msg).calibrated or self.arm_control_state != 'calibrating':
+            return
+
+        if self.arm_type == self.ArmType.RA:
+            raMotorsData = [0, 1, 0, 0, 0, 0]

```

Comment:
have you guys verified 1 goes up instead of -1?

Reply:
also is there a particular reason teleop is handling the calibration sequence instead of esw?

Reply:
Jetson teleop 1 is up, yes